### ISSUE TITLE
Wheel Picker not working inside scrollbar Android

### ISSUE THREAD

**Issue by swapnikap** [SuperRabbitD/NumberPicker#6]
Wheel Picker not working inside scrollbar Android

**Comment by SuperRabbitD** [comment]
Checking....

**Comment by mgursch** [comment]
It is not usable within a ScrollView - handling touch events wrong

**Comment by AlejandroHCruz** [comment]
Yes, it also doesn't work for me when using inside an `AnimatedExpandableListAdapter`. 
I think what has to be done is to make WheelPicker extend [NestedScrollView](https://developer.android.com/reference/androidx/core/widget/NestedScrollView) instead of extending the View class.

[This post](https://stackoverflow.com/questions/29885067/how-to-implement-nestedscrolling-on-android) on StackOverflow could result of interest as well.
